### PR TITLE
fix(Constants): allow undefined size

### PR DESCRIPTION
**Pull request by ImRodry**
**Please describe the changes this PR makes and why it should be merged:**
Fixes a bug accidentally introduced by #6637 where an error would be thrown when passing an undefined size (this is what happens when you only test the behavior you're looking for and not other scenarios 😅)

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by GoldenAngel2 — APPROVED** [review]
👍

File: src/util/Constants.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
 const AllowedImageSizes = Array.from({ length: 9 }, (e, i) => 2 ** (i + 4));
 
 function makeImageUrl(root, { format = 'webp', size } = {}) {
-  if (typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
+  if (size && typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
```
Comment (vladfrangu) [inline src/util/Constants.js:22]:
```suggestion
  if (typeof size !== 'undefined' && typeof size !== 'number') throw new TypeError('INVALID_TYPE', 'size', 'number');
```

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/util/Constants.js (hunk 1 above)
Comment (ImRodry) [reply src/util/Constants.js:22]: Does this work too? Looks cleaner imo
```suggestion
  if (!['undefined', 'number'].includes(typeof size)) throw new TypeError('INVALID_TYPE', 'size', 'number');
```

Comment (vladfrangu) [reply src/util/Constants.js:22]: Sure, that works too

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**